src/tests/issues.targets

```diff
@@ -731,10 +731,7 @@
         <ExcludeList Include="$(XunitTestBinBase)/GC/API/WeakReference/Finalize2/*">
             <Issue>Expectations about finalization order</Issue>
         </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/ilasm/PortablePdb/IlasmPortablePdbTests/*">
-            <Issue>Specific to CoreCLR</Issue>
-        </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/ilasm/System/Runtime/CompilerServices/MethodImplOptionsTests/*">
+        <ExcludeList Include="$(XunitTestBinBase)/ilasm/*">
```

Comment:
Do you know why these new exclusions don't work? NativeAOT outerloop is broken.

Also moving around MethodImplOptionsTests broke things.

https://github.com/dotnet/runtime/pull/107347/checks?check_run_id=29723577420

Reply:
I don't know why the exclusions aren't working... I should just mark the test as NativeAotIncompatible as it's not testing anything on NativeAOT in the first place.

The test is probably failing due to folder layout or something.